GitOut/Features/Wpf/Commands/Application.cs

```diff
@@ -0,0 +1,73 @@
+using System;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using System.Windows;
+using GitOut.Features.Material.Snackbar;
+
+namespace GitOut.Features.Wpf.Commands
+{
+    public class Application
+    {
+        private readonly ISnackbarService snack;
+
+        public Application(ISnackbarService snack)
+        {
+            this.snack = snack;
+
+            Open = new CompositeCommand<string>(OnOpen);
+            Copy = new CompositeCommand<string>(OnCopy);
+            RevealInExplorer = new CompositeCommand<string>(OnRevealInExplorer);
+        }
+
+        public static CompositeCommand<string>? Open { get; private set; }
+        public static CompositeCommand<string>? Copy { get; private set; }
+        public static CompositeCommand<string>? RevealInExplorer { get; private set; }
+
+        private void OnOpen(string? path)
+        {
+            if (path is not null)
```

Comment:
If we don't want null, can we instead not take a nullable string as parameter? Or is that because of some interface or something? :thinking:

Reply:
It's compositecommand that enforces it via the ExecuteAsync method 😒 
I wonder if CC can be tweaked though 🤔